webapp/views/wpt-app.js

```diff
@@ -340,6 +375,32 @@ class WPTApp extends PathInfo(WPTFlags(TestRunsUIBase)) {
     }
     return msg;
   }
+
+  computeBSFBannerMessage(isBSFCollapsed) {
+    if (isBSFCollapsed) {
+      return 'Browser Specific Failures graph (click the arrow to expand)';
+    }
+    return 'Browser Specific Failures graph (click the arrow to collapse)';
+  }
+
+  computeShowBSFGraph(pathIsRootDir, showBSF) {
+    return pathIsRootDir && showBSF;
```

Comment:
Worth a short comment string, something like:

```
    // Currently we only have BSF data for the entirety of the WPT test suite. To avoid
    // confusing the user, we only display the graph when they are looking at top-level
    // test results and hide it when in a subdirectory.
```

Reply:
Thanks!